spec/specified_js_runner_spec.vim

```diff
@@ -0,0 +1,32 @@
+source spec/support/helpers.vim
+
+describe "Runner"
+
+  before
+    cd spec/fixtures/jest
```

Comment:
Update to `cd spec/fixtures/js_multiple`

Reply:
get it